Hippo.Web/ClientApp/src/Shared/ShowFor.tsx

```diff
@@ -51,13 +51,23 @@ export const ShowFor = (props: Props) => {
 
   // cluster admins can access anything in their cluster
   if (
+    roles.some((r) => r === "ClusterAdmin") &&
     context.user.permissions.some(
       (p) => p.cluster === cluster && p.role === "ClusterAdmin",
     )
   ) {
     return <>{children}</>;
   }
```

Comment:
I don't think the comment, about cluster admins accessing anything, is still applicable with this change.